src/Glpi/Form/QuestionType/QuestionTypeDropdown.php

```diff
@@ -247,23 +247,24 @@ public function renderEndUserTemplate(
                 values,
                 '',
                 {
-                    'no_label': true,
-                    'values'  : checked_values,
-                    'multiple': is_multiple,
-                    'mb'      : '',
+                    'no_label'  : true,
+                    'values'    : checked_values,
+                    'multiple'  : is_multiple,
+                    'mb'        : '',
+                    'aria_label': label,
                 }
             ) }}
 TWIG;
 
         $twig = TemplateRenderer::getInstance();
-        $values = array_map(fn ($option) => $option['value'], $this->getValues($question));
         $checked_values = array_map(
-            fn ($option) => $option['value'],
+            fn ($option) => $option['uuid'],
```

Comment:
Main fix, send the uuid to be used as the dropdown 'selected' value, not the label.